checkstyle/import-control-metadata.xml

```diff
@@ -123,6 +123,7 @@
         <allow pkg="org.apache.kafka.server.util" />
         <allow pkg="org.apache.kafka.snapshot" />
         <allow pkg="org.apache.kafka.test" />
+        <allow pkg="org.apache.kafka.server.config" />
```

Comment:
Nit: The list is roughly ordered, can we put `org.apache.kafka.server.config` above?

Reply:
Fixed